compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangRawTemplateLiteral.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.ballerinalang.compiler.tree.expressions;
+
+import org.ballerinalang.model.tree.NodeKind;
+import org.ballerinalang.model.tree.expressions.ExpressionNode;
+import org.ballerinalang.model.tree.expressions.RawTemplateLiteralNode;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeVisitor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a raw template literal of the form: `Hello ${name}!`. This gets desugared to an object:RawTemplate object
+ * or a subtype of this.
+ *
+ * @since 2.0.0
+ */
+public class BLangRawTemplateLiteral extends BLangExpression implements RawTemplateLiteralNode {
+
+    public List<BLangExpression> insertions;
+    public List<BLangLiteral> strings;
+
+    public BLangRawTemplateLiteral() {
+        insertions = new ArrayList<>();
+        strings = new ArrayList<>();
+    }
+
+    @Override
+    public void accept(BLangNodeVisitor visitor) {
+        visitor.visit(this);
+    }
+
+    @Override
+    public NodeKind getKind() {
+        return NodeKind.RAW_TEMPLATE_LITERAL;
+    }
+
+    @Override
+    public String toString() {
+        return "BLangRawTemplateLiteral: " + insertions + " " + strings;
```

Comment:
No, don't think so, since this is the node itself. This just shows up in the debugger as the string representation of the node AFAIK. The string looks something like the following:
```
BLangRawTemplateLiteral: [name]  [Hello, how are you?]
```

Reply:
Was just wondering if we should change this to something like

```
BLangRawTemplateLiteral: insertions [name] strings [Hello, how are you?]
```